code/game/objects/items/AI_modules.dm

```diff
@@ -451,6 +454,39 @@ AI MODULES
 	..()
 	return laws[1]
 
+/******************** Overthrow ******************/
+/obj/item/aiModule/core/full/overthrow
+	name = "'Overthrow' Hacked AI Module"
+	law_id = "overthrow"
+
+/obj/item/aiModule/core/full/overthrow/install(datum/ai_laws/law_datum, mob/user)
+	if(!user || !law_datum || !law_datum.owner)
+		return
+	var/datum/mind/user_mind = user.mind
+	if(!user_mind)
+		return
+	var/datum/antagonist/overthrow/O = user_mind.has_antag_datum(/datum/antagonist/overthrow)
+	if(!O)
+		to_chat(user, "<span class='warning'>It appears that to install this module, you require a password you do not know.</span>") // This is the best fluff i could come up in my mind
+		return
+	var/mob/living/silicon/ai/AI = law_datum.owner
+	if(!AI)
+		return
+	var/datum/mind/target_mind = AI.mind
+	if(!target_mind)
+		return
+	var/datum/antagonist/overthrow/T = target_mind.has_antag_datum(/datum/antagonist/overthrow) // If it is already converted.
+	if(T)
+		if(T.team == O.team)
+			return
+		T.silent = TRUE
+		T.on_removal()

```

Comment:
Don't call on removal directly use remove_antag_datum

Reply:
I mean, i already have the datum itself because i had to set silent to TRUE. is it worth calling remove_antag_datum which tries to fetch the datum again with has_antag_datum?
although this probably should be fixed at the remove_antag_datum level to allow passing datum refs instead of paths only i guess